ui/round/src/plugins/keyboardMove.ts

```diff
@@ -1,5 +1,5 @@
 import { Dests } from '../interfaces';
-import { sanWriter, SanToUci } from './sanWriter';
+import { sanWriter, SanToUci } from 'nvui/sanWriter';
```

Comment:
it's weird to have an nvui dependency here.

Reply:
True. Is there a better place for sanWriter? Maybe ui/chess, as it deals with notation?

Reply:
yes

Reply:
Done. And praise rollup: when I checked, even with [glob imports](https://github.com/ornicar/lila/blob/b21eef0124194c7dc400cbd2a4b24a8a361c38e0/ui/analyse/src/ctrl.ts#L2), sanWriter and friends get stripped when unused.